tests/integration/states/file.py

```diff
@@ -209,6 +254,7 @@ def test_managed(self):
         self.assertEqual(master_data, minion_data)
         self.assertSaltTrueReturn(ret)
 
+    @skipIf(IS_WINDOWS, 'Mode not available in Windows')
```

Comment:
These tests should not be skipped. Instead, they should be modified that so that if we're running on Windows, we test to ensure that states return an appropriate error message instead of just exploding.